lib/foreman_hooks/util.rb

```diff
@@ -44,8 +44,12 @@ def exec_hook_int(stdin_data, *args)
       Open3.capture2e(*args.push(:stdin_data => stdin_data))
     else  # 1.8
       Open3.popen3(*args) do |stdin,stdout,stderr|
-        stdin.write(stdin_data)
-        stdin.close
+        begin
+          stdin.write(stdin_data)
+          stdin.close
+        rescue Errno::EPIPE
+          logger.debug "Input data not passed"
```

Comment:
This message is too short and contextless to be useful I think, it should give more information about what "data" is and what it's "passing" to - it won't be clear that this is from foreman_hooks.